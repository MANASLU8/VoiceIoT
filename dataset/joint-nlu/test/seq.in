тут слишком прохладно
установи температура на число а влажность на число
карта температур
что здесь так душно
если на улице больше число температура включи электроотопление
если на улице больше количество градуса включи отопление
когда будет готова пицца
когда будет готова еда в духовке
включена ли духовка
увеличь звук на три пункта
сделай громче
если будет дождь то сообщить взять зонт с уходом
увеличьте громкость чтобы не мешал стук
если я засну разбуди меня в число
отключи песню если я лягу
включай аудиосистему если я возвращаюсь домой
отключи музыку если я лягу
ставь эту песню чаще
кот давненько не ел покорми его
насыпай коту меньше корма
если на улице холодно включи батарею
закрыть дверь если холодно
включай отопление в комнате
закрыть окно если начался дождь
если на улице жарко открой окно
проветрите сейчас пока я буду на кухне
если станет холодно то открой окно
приготовь хлеб к моему приходу
если за окном рассвет открой занавески
когда последний раз открывали шторы
если за окном солнце закрой занавески
прикрой шторы если слишком светло
откроешь шторы если слишком темно
ежели за окном солнце открой шторы
включай свет когда я захожу в гостиную
выключай свет если я ухожу из комнаты
выключай свет когда я выйду из спальной
включай свет опять я зашёл в спальню
какие двери в доме открыты
какие из окон в квартире сейчас открыты
если будет жарко значит открой окно
меняй каждый день фонарик на время
выключай свет когда я иду из санузла
выключай свет если я выхожу из спальни
включай свет когда я захожу в гостиную
сделай кофе и чай
когда я зайду сделай чай
доложи о соcтоянии одежды
постирай на 90 градусах
подмети там пока я буду на кухне
подмети здесь пока я буду на кухне
запиши передачу если я засну
включи телевизор
запомни передачу если я лягу
предупреди если автомобиль сзади меня придется очень вплотную
предупреди когда грузовик сзади меня будет вплотную
ктонибудь был дома сегодня днем
сколько дней стоит салат в холодильнике
какие у нас есть орехи
свежесть супа
сделай кофе к моему приходу
каждый день проверяй кофеварку в доме
сколько в этом году я заплатил за свет
выключай свет когда я выйду из комнаты
выключи электричество в ванной
сделай напор сильнее
наполни горячую ванную к моему приходу
я собираюсь принять ванну
я хочу принять ванну
включи утюг я хочу погладить шелковую блузку
мне придется доехать до число больницы
мне надо добраться до число поликлиники
распечатай документ название через число секунд
распечатай паспорт название через число часа