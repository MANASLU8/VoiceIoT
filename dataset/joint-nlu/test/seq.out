B-TemperatureSensor.LOC B-TemperatureSensor.IntensityQuantifier B-TemperatureSensor.CommandFeature
B-TemperatureSensor.COMMAND B-TemperatureSensor.CommandParameter O B-TemperatureSensor.CommandParameterValue O B-TemperatureSensor.CommandParameter O B-TemperatureSensor.CommandParameterValue
B-TemperatureSensor.DeviceFeature I-TemperatureSensor.DeviceFeature
B-TemperatureSensor.ParameterValueQuestion B-TemperatureSensor.LOCCurrent B-TemperatureSensor.IntensityQuantifier B-TemperatureSensor.CommandFeature
O B-TemperatureSensor.LOC I-TemperatureSensor.LOC O B-TemperatureSensor.CommandParameterValue B-TemperatureSensor.CommandParameter B-TemperatureSensor.COMMAND B-TemperatureSensor.DEVICE
O B-TemperatureSensor.LOC I-TemperatureSensor.LOC B-TemperatureSensor.Quantor B-TemperatureSensor.CommandParameterValue B-TemperatureSensor.CommandParameter B-TemperatureSensor.COMMAND B-TemperatureSensor.DEVICE
B-SmartOven.ParameterValueQuestion B-SmartOven.FeatureState I-SmartOven.FeatureState B-SmartOven.CommandFeature
B-SmartOven.CommandQuestion B-SmartOven.FeatureState I-SmartOven.FeatureState B-SmartOven.CommandFeature O B-SmartOven.DEVICE
B-SmartOven.DeviceSta`te O B-SmartOven.DEVICE
B-AudioSystem.COMMAND B-AudioSystem.CommandFeature O B-AudioSystem.DeviceParameterValue B-AudioSystem.DeviceFeature
B-AudioSystem.COMMAND B-AudioSystem.IntensityQuantifier
O O O O B-AudioSystem.COMMAND I-AudioSystem.COMMAND B-AudioSystem.CommandFeature O B-AudioSystem.Action
B-AudioSystem.COMMAND B-AudioSystem.CommandFeature O O O O
O B-AudioSystem.Actor B-AudioSystem.Action B-AudioSystem.COMMAND O O B-AudioSystem.DeviceParameterValue
B-AudioSystem.COMMAND B-AudioSystem.CommandFeature O B-AudioSystem.Actor B-AudioSystem.Action
B-AudioSystem.COMMAND B-AudioSystem.DEVICE O O O O
B-AudioSystem.COMMAND B-AudioSystem.CommandFeature O B-AudioSystem.Actor B-AudioSystem.Action
B-AudioSystem.COMMAND O B-AudioSystem.CommandFeature B-AudioSystem.IntensityQuantifier
O O O O B-SmartFeeder.COMMAND O
B-SmartFeeder.COMMAND B-SmartFeeder.CommandFeature B-SmartFeeder.IntensityQuantifier B-SmartFeeder.DeviceFeature
O O B-HeaterController.LOC B-HeaterController.CommandFeature B-HeaterController.COMMAND B-HeaterController.DeviceFeature
B-HeaterController.COMMAND B-HeaterController.CommandFeature O O
B-HeaterController.COMMAND B-HeaterController.DeviceFeature O B-HeaterController.LOC
B-HumiditySensor.COMMAND B-HumiditySensor.DeviceFeature O O O
O O O O B-WindowRelay.COMMAND B-WindowRelay.CommandFeature
B-WindowRelay.COMMAND O O B-WindowRelay.Actor B-WindowRelay.Action O B-WindowRelay.LOC
O O O O B-WindowRelay.COMMAND B-WindowRelay.DeviceFeature
B-SmartBreadMaker.COMMAND B-SmartBreadMaker.CommandFeature O B-SmartBreadMaker.Actor B-SmartBreadMaker.Action
O O O O O O
B-CurtainsRelay.ParameterValueQuestion O O B-CurtainsRelay.CommandFeature I-CurtainsRelay.CommandFeature
O O O O O O
B-LightSensor.COMMAND B-LightSensor.CommandFeature B-LightSensor.LOC B-LightSensor.IntensityQuantifier B-LightSensor.CommandFeature
B-LightSensor.COMMAND B-LightSensor.CommandFeature B-LightSensor.LOC B-LightSensor.IntensityQuantifier B-LightSensor.CommandFeature
O B-LightSensor.LOC I-LightSensor.LOC B-LightSensor.CommandFeature B-LightSensor.COMMAND B-LightSensor.CommandFeature
B-LightSensor.COMMAND B-LightSensor.CommandFeature O B-LightSensor.Actor B-LightSensor.Action O B-LightSensor.LOC
B-LightSensor.COMMAND B-LightSensor.CommandFeature O B-LightSensor.Actor B-LightSensor.Action O B-LightSensor.LOC
B-LightSensor.COMMAND B-LightSensor.CommandFeature O B-LightSensor.Actor B-LightSensor.Action O B-LightSensor.LOC
B-LightSensor.COMMAND B-LightSensor.CommandFeature O B-LightSensor.Actor B-LightSensor.Action O B-LightSensor.LOC
B-DoorWindowStatusSensor.RequestQuestion B-DoorWindowStatusSensor.DeviceFeature O B-DoorWindowStatusSensor.LOC B-DoorWindowStatusSensor.FeatureState
B-DoorWindowStatusSensor.DeviceQuestion O B-DoorWindowStatusSensor.DeviceFeature O B-DoorWindowStatusSensor.LOC O B-DoorWindowStatusSensor.FeatureState
O O B-DoorWindowStatusSensor.CommandFeature O B-DoorWindowStatusSensor.COMMAND B-DoorWindowStatusSensor.CommandFeature
B-Lamp.COMMAND O O B-Lamp.DeviceFeature O B-Lamp.CommandParameterValue
B-Lamp.COMMAND B-Lamp.CommandFeature O B-Lamp.Actor B-Lamp.Action O B-Lamp.LOC
B-Lamp.COMMAND B-Lamp.CommandFeature O B-Lamp.Actor B-Lamp.Action O B-Lamp.LOC
B-Lamp.COMMAND B-Lamp.CommandFeature O B-Lamp.Actor B-Lamp.Action O B-Lamp.LOC
B-SmartKettle.COMMAND B-SmartKettle.CommandFeature O I-SmartKettle.CommandFeature
O B-SmartKettle.Actor B-SmartKettle.Action B-SmartKettle.COMMAND B-SmartKettle.CommandFeature
O O B-WashingMachine.CommandState B-WashingMachine.CommandFeature
B-WashingMachine.COMMAND O B-WashingMachine.DeviceParameterValue B-WashingMachine.DeviceParameter
B-SmartVacuumCleaner.COMMAND B-SmartVacuumCleaner.LOC O B-SmartVacuumCleaner.Actor B-SmartVacuumCleaner.Action O B-SmartVacuumCleaner.LOC
B-SmartVacuumCleaner.COMMAND B-SmartVacuumCleaner.LOCCurrent O B-SmartVacuumCleaner.Actor B-SmartVacuumCleaner.Action O B-SmartVacuumCleaner.LOC
B-TVSet.COMMAND B-TVSet.Devicefeature O B-TVSet.Actor B-TVSet.Action
B-TVSet.COMMAND B-TVSet.DEVICE
B-TVSet.COMMAND B-TVSet.DeviceFeature O B-TVSet.Actor B-TVSet.Action
B-MotionSensor.COMMAND O O O O O O O
B-MotionSensor.COMMAND O B-MotionSensor.CommandFeature B-MotionSensor.LOC I-MotionSensor.LOC B-MotionSensor.FeatureState B-MotionSensor.IntensityQuantifier
B-MotionSensor.Actor B-MotionSensor.Action B-MotionSensor.LOC O O
B-SmartFridge.RequestQuestion O B-SmartFridge.FeatureState B-SmartFridge.CommandFeature O B-SmartFridge.LOC
B-SmartFridge.ParameterValueQuestion O O B-SmartFridge.CommandFeature I-SmartFridge.CommandFeature
B-SmartFridge.DeviceFeature I-SmartFridge.DeviceFeature
B-CoffeeMachine.COMMAND B-CoffeeMachine.Ð¡ommandFeature O B-CoffeeMachine.Actor B-CoffeeMachine.Ation
O O B-CoffeeMachine.COMMAND B-CoffeeMachine.DEVICE O B-CoffeeMachine.LOC
B-SmartElectricityMeter.ParameterValueQuestion O O O B-SmartElectricityMeter.Actor B-SmartElectricityMeter.Action O O
B-SmartElectricityMeter.COMMAND B-SmartElectricityMeter.CommandFeature O B-SmartElectricityMeter.Actor B-SmartElectricityMeter.Action O B-SmartElectricityMeter.LOC
B-SmartElectricityMeter.COMMAND B-SmartElectricityMeter.CommandFeature O B-SmartElectricityMeter.LOC
B-WaterTapRelay.COMMAND B-WaterTapRelay.CommandFeature B-WaterTapRelay.Quantifier
B-WaterTapRelay.COMMAND B-WaterTapRelay.FeatureState B-WaterTapRelay.CommandFeature O B-WaterTapRelay.Actor B-WaterTapRelay.Action
O B-WaterTapRelay.Actor B-WaterTapRelay.Action I-WaterTapRelay.Action
B-WaterTapRelay.Actor B-WaterTapRelay.Action I-WaterTapRelay.Action B-WaterTapRelay.CommandFeature
B-SmartIron.COMMAND B-SmartIron.DEVICE B-SmartIron.Actor B-SmartIron.Action I-SmartIron.Action B-SmartIron.DeviceParameterValue O
B-SmartCar.Actor B-SmartCar.COMMAND I-SmartCar.COMMAND O B-SmartCar.CommandParameterValue B-SmartCar.LOC
B-SmartCar.Actor B-SmartCar.COMMAND I-SmartCar.COMMAND O B-SmartCar.CommandParameterValue B-SmartCar.LOC
B-Printer.COMMAND B-Printer.CommandFeature I-Printer.CommandFeature O O O
B-Printer.COMMAND B-Printer.CommandFeature I-Printer.CommandFeature O O O